formatTest/unit_tests/expected_output/modules.re

```diff
@@ -608,3 +608,6 @@ let foo =
       module Y: Y_t with type t = a,
       module Z: Z_t with type t = a,
     ) => X.a;
+
+/* https://github.com/facebook/reason/issues/2028 */
+M.([]);
```

Comment:
do we need the parens?

Reply:
I'm not sure. I'll investigate if there are conflicts.